src/lib/providers/BidProvider.tsx

```diff
@@ -126,13 +130,14 @@ const BidProvider: React.FC<BidProviderType> = ({ children }) => {
 	};
 
 	const getBidsForAccount = async (id: string) => {
+		if (baseApiUri === undefined) {
+			throw Error(`no api endpoint`);
+		}
 		try {
-			const bids = await zAuction.getBidsForAccount(id);
+			const bids = await zAuction.getBidsForAccount(baseApiUri!, id);
```

Comment:
You shouldn't have to use `!` here if you did the check above to make sure it is not undefined.